### DISCUSSION TITLE
text underline

### DISCUSSION THREAD

**Discussion by Russo-creation** [protectwise/troika#268] [Q&A]
Hi, 

Is there any option to make text with underline - I know that there are some kind of font's that has underline like https://www.fonts101.com/fonts/view/Uncategorized/22804/Labtop_Underline but could I make underline for arial font?

**Comment by lojjic** [comment]
Would you want the entire string of text underlined or only a portion?

Either way it's not currently a supported feature, but you could probably use [getSelectionRects](https://protectwise.github.io/troika/troika-three-text/#carets-and-selection-ranges) and maybe some of the other data in `text.textRenderInfo` to create underline plane meshes yourself...?

**Comment by Russo-creation** [comment]
@lojjic thanks for the answer. I was thinking about typical usage of underline like it is done through the css `text-decoration: underline;` and for multiple lines
